std/conv.d

```diff
@@ -178,14 +167,14 @@ $(I UnsignedInteger):
 template to(T)
 {
     T to(A...)(A args)
-        if (!isRawStaticArray!A)
+        if (A.length > 0 && !isStaticArray!(A[0]))
```

Comment:
I believe the previous constraint is equivalent to this instead:

```D
if(A.length > 0 || !isStaticArray!(A[0]))
```

The calls it allows vs. your constraint, I'm not sure are valid. But this is a difference.